### PR TITLE
chore: add k8 priority change caveat [DET-7212]

### PR DESCRIPTION
**Pull request by hamidzr**
## Description

changing priority, even increasing it, on Kubernetes has to be followed by a pod termination which can be counterintuitive to users. We add a message about it to the UI here.

## Commentary (optional)
in the priority section for updating a k8 job:
Screen Shot 2022-04-22 at 15 07 21

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/features/job-queue.txt [hunk 1]
```diff
@@ -137,3 +137,6 @@ There are a number of constraints associated with using the job queue to modify
 
 -  The change resource pool operation can only be performed on experiments. To change the resource
    pool of other tasks, cancel the task and resubmit it.
+
+-  Changing priority in Kubernetes always involves an implicit cancleation of the job, even in
```
Comment (julian-determined-ai) [inline docs/features/job-queue.txt:141]:
```suggestion
-  Changing priority in Kubernetes always involves an implicit cancellation of the job, even in
```

**Review by julian-determined-ai — APPROVED** [review]
One typo update other looks good!

**Comment by eecsliu** [comment]
Pulling this in with the kubernetes changes: https://github.com/determined-ai/determined/pull/3938

**Comment by hamidzr** [comment]
> Pulling this in with the kubernetes changes: #3938

I don't think this one is the one you want. you probably want https://github.com/determined-ai/determined/pull/4000

**Review by eecsliu — APPROVED** [review]
🚢

**Comment by hamidzr** [comment]
added through #4000

**Pull request closed without merging**